awx/ui_next/src/screens/Job/JobOutput/JobOutput.jsx

```diff
@@ -105,11 +143,22 @@ class JobOutput extends Component {
     this.isRowLoaded = this.isRowLoaded.bind(this);
     this.loadMoreRows = this.loadMoreRows.bind(this);
     this.scrollToRow = this.scrollToRow.bind(this);
+    this.monitorJobSocketCounter = this.monitorJobSocketCounter.bind(this);
   }
 
   componentDidMount() {
+    const { job } = this.props;
     this._isMounted = true;
     this.loadJobEvents();
+
+    connectJobSocket(job, data => {
+      if (data.counter && data.counter > this.jobSocketCounter) {
+        this.jobSocketCounter = data.counter;
+      } else if (data.final_counter && data.unified_job_id === job.id) {
+        this.jobSocketCounter = data.final_counter;
```

Comment:
The only thing _actually_ getting updated here is the total count, which lets react-virtualize know to expand it's row range.

In another PR, it would be good to actually put the received output data in a cache that we can check prior to making api requests for the events.